src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -63,6 +63,39 @@ function createHealthCareRegionListPages(page, drupalPagePath, files) {
     page,
     'health_care_region_locations_page.drupal.liquid',
   );
+
+  const relatedLinks = { fieldRelatedLinks: page.fieldRelatedLinks };
+
+  // Create the detail page for healthcare local facilities
+  if (page.mainFacilities !== undefined || page.otherFacilities !== undefined) {
+    for (const facility of page.mainFacilities.entities) {
+      if (facility.entityBundle === 'health_care_local_facility') {
+        const mainFacilityCompiled = Object.assign(facility, relatedLinks);
+        files[
+          `drupal${drupalPagePath}/locations/${
+            facility.fieldFacilityLocatorApiId
```

Comment:
i think this needs to use the path from the entityUrl attribute, or somehow url-ify the facility name using dashes instead of using the locator api id. same for line 90 below

Reply:
Either works for me, but I think the Facility Locator right now uses `fieldFacilityLocatorApiId` in the URL so I think this is safe.

Reply:
per content team we should be using facility nickname for the path so I made this change.